dash/development/base_component.py

```diff
@@ -132,13 +134,26 @@ def __init__(self, **kwargs):
                     f"\nAllowed arguments: {allowed_args}"
                 )
 
-            if k != "children" and isinstance(v, Component):
-                raise TypeError(
-                    error_string_prefix
-                    + " detected a Component for a prop other than `children`\n"
-                    + "Did you forget to wrap multiple `children` in an array?\n"
-                    + f"Prop {k} has value {v!r}\n"
```

Comment:
hmph and right after @chriddyp just upgraded this error message #2027 - I suppose an advantage of building the `_children_props` list into the Python component generator is we could keep this error, just qualify it further to
```
if k != "children" and k not in self._children_props and isinstance(v, Component):
```